x-pack/plugins/fleet/common/types/rest_spec/health_check.ts

```diff
@@ -7,12 +7,15 @@
 
 export interface PostHealthCheckRequest {
   body: {
-    host: string;
+    id: string;
+    // deprecated
```

Comment:
This way it shows up as deprecated in the IDE 
```suggestion
    /** @deprecated use id field instead */
```